oceanwaters/doc/setup_dev_env.md

```diff
@@ -1,99 +1,54 @@
-Requirements
-============
+These instructions cover system requirements, and the installation of
+prerequisite software (PLEXIL, ROS), for OceanWATERS.  Familiarity with
+Unix/Linux and the Bash shell command line is assumed.
 
-Hardware
---------
+System Requirements
+-------------------
 
-The minimum recommended hardware features are as follows:
-* 2.0 GHz quad core x86-64 CPU
-* 16 GB memory
-* Nvidia graphics card. GeForce GTX 750-Ti, GeForce GTX 960M, Quadro M1200, or better.
+### Hardware
 
-Software
---------
+The minimum recommended hardware configuration is as follows:
 
-The following software is required to build and run OceanWATERS:
-* Operating System: Linux Ubuntu 18.04 (bionic) specifically. (Even slightly
-newer versions such as 18.10 may not work.)
-* The Bash command shell
-* Robot Operating System (ROS), melodic version specifically, including:
-* Gazebo simulator 9.13
-* catkin build system
-* PLEXIL plan language and executive (http://plexil.sourceforge.net).
+-   2.0 GHz quad core x86-64 CPU
+-   16 GB memory
+-   Nvidia graphics card. GeForce GTX 750-Ti, GeForce GTX 960M, Quadro M1200, or
+    better.
 
-Note on virtual machines (e.g. VMWare, Parallels, VirtualBox, Windows Subsystem
-for Linux): These have all worked to some degree, but tend to not support the
-Gazebo simulator very well or at all. They are not recommended for OceanWATERS.
+### Software
 
-Development Environment Setup
-=================
+The following minimum software set is required to build and run OceanWATERS
+(version numbers and identifiers are listed where specific versions are
+required):
 
-ROS and Gazebo
-------------
+-   The Ubuntu 18.04 (Bionic) Linux operating system, with:
+    -   The Bash command shell
+    -   The git version control system
+-   The Robot Operating System (ROS), Melodic Morenia distribution, with:
+    -   Gazebo 9.13
```

Comment:
Gazebo is not part of ROS and should be listed at the same bullet level.

Reply:
I think in all my many previous installations of the system, I got Gazebo (albeit the wrong version) when I was following instructions to install "ROS".  Nevertheless, I have made this edit.